code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -464,3 +464,21 @@
 	can_hold = list(
 		/obj/item/ammo_casing/afoam_dart
 		)
+
+/obj/item/storage/belt/plant
+	name = "botany toolbelt"
+	desc = "A belt to hold various botanical tools."
+	icon_state = "plantbelt"
+	storage_slots = 8
+	can_hold = list(
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/glass/beaker,
+		/obj/item/material/minihoe,
+		/obj/item/shovel/spade,
+		/obj/item/material/knife/machete/hatchet,
+		/obj/item/analyzer/plant_analyzer,
+		/obj/item/gun/energy/floragun,
+		/obj/item/tool/wirecutters,
+		/obj/item/tool/wrench,
+		/obj/item/multitool,

```

Comment:
What are these tools for?

Reply:
The floragun is the floral somatoray, for xenobotany, wirecutters function as improvised hedge trimmers, wrenches are used to move trays around, should you want to, and multitools are used to toggle cryogenic freezing on a plant

Reply:
Floragun I'm fine with (And don't think was a line I selected for the comment, just auto-shown).
"Improvised" means it doesn't need to be held by a belt dedicated to storing proper tools for a job. E.g., a surgery belt shouldn't be able to hold screwdrivers or welding tools.
I'm also dubious about whether the "move trays around" or "toggle cryogenic freezing" are something that'd be needed regularly enough to really be warranted. Mind, these tools have a _lot_ of other uses, compared to the botany tools listed.